content/working_with/manager/update-deployment.md

```diff
@@ -257,7 +257,7 @@ The `update` operations are:
 - `cloudify.interfaces.lifecycle.update_config`
 - `cloudify.interfaces.lifecycle.update_apply`
 
-Plugin authors can implement any, or all, of those operations. Those three operations can be written in a way to mirror the `create`, `configure`, and `start` operations.
+Plugin authors can implement any, or all, of those operations. Those three operations can be written in a way so that they mirror the `create`, `configure`, and `start` operations.
```

Comment:
This should probably be: "Plugin authors can implement any, or all, of these operations. These three operations can be written so that they mirror the ..."